taskcluster/fenix_taskgraph/transforms/browsertime.py

```diff
@@ -124,3 +124,18 @@ def fill_email_data(config, tasks):
             email["subject"] = email["subject"].format(**format_kwargs)
 
         yield task
+
+
+@transforms.add
+def filter_ytp_tests(config, tasks):
+    """Youtube-playback AV1, H264 and VP9 tests should run on P2 only"""
+    for task in tasks:
+        test_platform = task['test-platform']
+        test_name = task["test-name"]
+        if "AV1" in test_name or "H264" in test_name or "VP9" in test_name:
```

Comment:
H264 tests should be fine to run on G5, we have them running in raptor atm. You can remove that restriction: https://treeherder.mozilla.org/#/jobs?repo=mozilla-central&tier=1%2C2%2C3&searchStr=youtube&revision=2cca9cb89b46d67e1e25051495cb89f71b856c32